bin/deploy-vhost

```diff
@@ -469,17 +470,34 @@ sub create_or_update_files {
             "-C",
             "/tmp/",
         );
-        print "Moving extracted package to $vcspath_install...\n";
-        shell(
-            "mv",
-            "/tmp/$vhost",
-            "$vhost_dir/$vcspath_install",
-        );
-        print "Fixing file ownership...\n";
+        print "Changing file ownership to $conf->{'user'}...\n";
         shell(
             "chown",
             "-R",
             $conf->{'user'} . ':' . $conf->{'user'},
+            "/tmp/$vhost",
+        );
+        foreach (@{$conf->{packaging}->{custom_unpackage_locations} || []}) {
+            my $package_path = $_->{package_path};
+            if (!$package_path) {
+                die "Missing 'package_path' in 'custom_unpackage_locations' entry " . Dumper($_);
+            }
+            my $vhost_path = $_->{vhost_path};
+            if (!$vhost_path) {
+                die "Missing 'vhost_path' in 'custom_unpackage_locations' entry " . Dumper($_);
+            }
+            print "Moving $package_path to vhost path $vhost_path...\n";
+            my $source_path = "/tmp/$vhost/$package_path";
+            if (-d $source_path) {
+                $source_path .= "/";
+            }
```

Comment:
Is this check intended to support individual files? I think in most cases it will be a directory, and we'll want a trailing slash to make rsync work as we need it to in this case.